sycl/unittests/assert/assert.cpp

```diff
@@ -438,7 +438,7 @@ void ChildProcess(int StdErrFD) {
     exit(1);
   }
 
-  sycl::platform Plt{sycl::default_selector()};
+  sycl::platform Plt{sycl::default_selector_v};
```

Comment:
I think we can drop initialization list here and all places below in this file. https://github.com/intel/llvm/pull/6624 makes it unnecessary.

Reply:
Yes.